CODING_STANDARD

```diff
@@ -92,6 +92,34 @@ Header files:
   of the header file rather than in line
 - Guard headers with #ifndef CPROVER_DIRECTORIES_FILE_H, etc
 
+Make files
+- Each source file should appear on a separate line
+- The final source file should still be followed by a trailing slash
+- The last line should be a comment to not be deleted, i.e. should look like:
+```
+SRC = source_file.cpp \
+      source_file2.cpp \
+      # Empty last line
+```
+- This ensures the Makefiles can be easily merged.
```

Comment:
This sounds good - but it is sort of meaningless until they are all sorted (where as this change can come in to effect gradually I think). If someone wants to go through and fix this (probably with a script I guess...) then this sounds like a good rule. 

That said, I'm not sure I see a big benefit though, Makefiles are mainly for the computer rather than the developer to read.